Documentation.md

```diff
@@ -418,25 +361,528 @@ In order to set the right connector parameters, all you have to do is to declare
 ]
 </pre>
 
+### External bot
+Let your bot interact with online visitors directly within iAdvize’s chatbox.
+The External bot plugin enables iAdvize's Admins and Managers to create users with the role “bot” from iAdvize’s administration. The scenario and availability of the bot are managed by your app. 
 
-** Request - GET method **
+Tu put it in a nutshell, the External bot plugin:
+* Allows bots providers to create their connector thanks to the Developper Platform,
+* Allows customers to connect their iAdvize account to a bot provider and connect bots seamlessly. These bots interact with iAdvize like a human agent.
+
+#### The External bot user flow
+
+The iAdvize Administrator/Manager:
+
+1. Activates your connector from the iAdvize marketplace,
+2. Creates a new user with the role "bot" from the "People" section of iAdvize,
+3. Select an external bot and a scenario,
+4. is redirected to your bot edition page.
+
+![Bot plugin](./assets/images/interactions-bot-feature.png)
+
+**Add the External bot plugin**
+To make sure your connector uses the External Bot plugin correctly, all you have to do is to declare the base url that will be postfixed by seven endpoints described below.
```

Comment:
.... to declare : 
* the base url that will be postfixed by seven endpoints described below

(si on veut respecter la même forme que les autres interactions)